### ISSUE TITLE
starterator crashes with "not in list error"

### ISSUE THREAD

**Issue by cdshaffer** [cdshaffer/starterator#16]
OK running phage dori, starterater crashes on pham 4536. stack trace puts error at phams.py line 220:
```
most_called_start_index = self.total_possible_starts.index(called_starts_count[0][0])+1
```
error message says "93 not in list"

the gene for this pham is the last gene in the genome ORF : (94) Reverse 64210. .64530
also just to note: gene is an orpham so no other genes in pham

**Comment by cdshaffer** [comment]
Same error and stack trace for phage Snape. 26 of 100. 
."45 is not in list."

**Comment by cdshaffer** [comment]
I figured out for Dori-94 that the issue was that the start codon of the very last gene (reverse) was less than 99 bases from end of phage. The program was set to look at the upstream 99 bases and never checked to see if there *was* 99 bases upstream, added a line to ~113 of phamgene to take the smaller of the 99 bases upstream or the sequence all the way to the end of the phage. Also noticed in the following line that there was a +1 to the index of the "stop" site, but this was not needed to it was removed.  Starterator does not now crash with dori. still crashes with snape.
